data-browser/src/test/scala/hmda/dataBrowser/api/DataBrowserDirectivesSpec.scala

```diff
@@ -181,7 +181,8 @@ class DataBrowserDirectivesSpec extends WordSpec with ScalatestRouteTest with Ma
       val route: Route = failingRoute(extractMsaAndStateAndCountyAndInstitutionIdentifierBrowserFields)
 
       Get("/?msamds=34980&leis=BANK0&states=CA,AK&actions_taken=1,2,3&counties=19125&years=2018") ~> route ~> check {
-        responseAs[String].contains("provide-only-msamds-or-states-or-counties-or-leis") shouldBe true
+        responseAs[String].contains("A Valid LEI should be 20 characters and Alphanumeric") shouldBe true
```

Comment:
Update here to reflect a recent code change

Reply:
I've learned over time it is better to drive functionality (including test verification) with error and status codes instead of text messages. (In one extreme case, a certain company got its QA suite to a point where they could never allow error message changes...) One or two instances is probably not a big deal.